src/controllers/mainController.ts

```diff
@@ -870,16 +880,31 @@ export default class MainController implements vscode.Disposable {
 	/**
 	 * Prompt the user to view release notes if this is new extension install
 	 */
-	private async showReleaseNotesPrompt(): Promise<void> {
+	private async showFirstLaunchPrompts(): Promise<void> {
```

Comment:
Given that we're not even awaiting this call anyways (and we shouldn't be) I think it'd make more sense to just make this non-async and then leave a comment in the below calls that we're purposely not blocking on the toasts being handled so as not to block extension startup